### DISCUSSION TITLE
Is it possible to only add files in the current folder to the navigation?

### DISCUSSION THREAD

**Discussion by darkdragon-001** [mkdocs/mkdocs#2651] [Q&A]
Example:
```
FOLDER1
File1
File2
Subfolder1
  File3
Subfolder2
  File4
```

When `File4` is the current file, it should hide `File3`.

**Comment by oprypin** [comment]
As I understand, you're looking for a customization of how the nav looks, not to actually exclude files from the nav.

This depends entirely on the theme.
"readthedocs" theme apparently doesn't do that indeed, and it's an intentional design choice, there's no such option.

An external theme "mkdocs-material" does what you want by default, check it out.
